Include/Rover/Range.hpp

```diff
@@ -80,11 +80,15 @@ namespace Rover {
     private:
       Type calculate_random(const Type& begin, const Type& end);
       Type pick_random(const Type& lhs, const Type& rhs);
+      template<typename TypeFwd>
+      Type round(Evaluator& evaluator, TypeFwd&& value);
+      using GranularityPlaceholder = std::conditional_t<std::is_same_v<
```

Comment:
Place type declarations at the top, ie, ```using Granularity...``` goes on line 81.
Next list all member variables.
Next put one single blank line.
Next list all member functions, do not separate them with a blank line.

ie:

```
  private:
    using T1 = ...;
    using T2 = ...;
    int m_member1;

    void method1();
```